### DISCUSSION TITLE
"Github Explore the week" newsletter discontinued?

### DISCUSSION THREAD

**Discussion by darkred** [community/community#67875] [Other Feature Feedback, Questions, & Ideas] [answered]
I haven't received a "Github Explore the week" email newsletter since Sept. 5 (week of Aug 29 - Sep 5). 
It is discontinued? It was really useful.

Here is a screenshot of that last email:
![2023-09-21_205633](https://github.com/community/community/assets/723651/aa14b9c3-7227-4456-990e-d0c22d435e3b)

And here is a screenshot of https://github.com/settings/emails | Email preferences: 
![2023-09-28_223853](https://github.com/community/community/assets/723651/65fda4d8-d29f-4149-ab83-658d78b77d88)

---

(Update 9/26)
I didn't receive the weekly email today either.

For reference, the "Github Explore the week" email was sent every Tuesday. At the exact same time I was also receiving the weekly "GitHub security alert digest" email.
I currently only get the latter one ("GitHub security alert digest"). Today I got that for the week of Sep 19-Sep 26.

Does it occur to you too?

**Comment by sujjeee** [comment]
I subscribed to the daily newsletter, but I’m not getting any newsletters anymore. Why is that? The newsletter helped me a lot to learn and explore new tech stacks. How can I get the daily newsletter again on my email?

**Reply by NeckBeardPrince** [reply]
Same issue here

**Comment by joshuawalker** [comment, marked as answer]
Hey folks, the Explore weekly newsletter was turned off in September 2023 but you can still check out Explore at https://github.com/explore.
